GameDesign/Assets/Scripts/Managers/LevelSettingManager.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LevelSettingManager
+{
+    #region attributes
+    private static LevelSettingManager instance_;
+
+
+    public int CustomersToBeSpawnedWM { get; private set; }
+    public int[] CustomersWithOutMask { get; private set; }
```

Comment:
actually maybe was better. but I implemented it changing how it looks in the editor. so now for all intents and purposes is should be the same. but it can be changed later when doing the spawning. i just did it like this so we can edit it in the editor view, not sure if a map would work there